config/standalone-webpack.config.js

```diff
@@ -0,0 +1,288 @@
+
+const getAbsolutePath = relativePath =>
+  path.join(__dirname, '../', relativePath);
+
+const globalEntryFiles = {
+  polyfills: getAbsolutePath('src/platform/polyfills/preESModulesPolyfills.js'),
+  style: getAbsolutePath('src/platform/site-wide/sass/style.scss'),
+  styleConsolidated: getAbsolutePath(
+    'src/applications/proxy-rewrite/sass/style-consolidated.scss',
+  ),
+  vendor: [
+    getAbsolutePath('src/platform/polyfills'),
+    'react',
+    'react-dom',
+    'react-redux',
+    'redux',
+    'redux-thunk',
+    '@sentry/browser',
+  ],
+};
+
+/**
+ * Get a list of all the entry points.
+ *
+ * If the `entry` CLI argument is passed, only the specified
+ * application entries are built.
+ */
+function getEntryPoints(entry) {
+  const manifests = getAppManifests(path.join(__dirname, '../'));
+  let manifestsToBuild = manifests;
+  if (entry) {
+    const entryNames = entry.split(',').map(name => name.trim());
+    manifestsToBuild = manifests.filter(manifest =>
+      entryNames.includes(manifest.entryName),
+    );
+  }
+
+  return getWebpackEntryPoints(manifestsToBuild);
+}
+
+module.exports = env => {
+  const buildOptions = Object.assign(
+    {},
+    {
+      buildtype: 'localhost',
+      api: '',
+    },
+    env,
+  );
+  // Assign additional defaults which reference other properties
+  Object.assign(buildOptions, {
+    destination: path.resolve('../', 'build', buildOptions.buildtype),
+  });
+  const apps = getEntryPoints(buildOptions.entry);
+  const entryFiles = Object.assign({}, apps, globalEntryFiles);
+  const isOptimizedBuild = [
+    ENVIRONMENTS.VAGOVSTAGING,
+    ENVIRONMENTS.VAGOVPROD,
+  ].includes(buildOptions.buildtype);
+
+  // enable css sourcemaps for all non-localhost builds
+  // or if build options include local-css-sourcemaps or entry
+  const enableCSSSourcemaps =
+    buildOptions.buildtype !== ENVIRONMENTS.LOCALHOST ||
+    buildOptions['local-css-sourcemaps'] ||
+    !!buildOptions.entry;
+
+  const baseConfig = {
+    mode: 'development',
+    entry: entryFiles,
+    output: {
+      path: `${buildOptions.destination}/generated`,
+      publicPath: '/generated/',
+      filename: !isOptimizedBuild
+        ? '[name].entry.js'
+        : `[name].entry.[chunkhash]-${timestamp}.js`,
+      chunkFilename: !isOptimizedBuild
+        ? '[name].entry.js'
+        : `[name].entry.[chunkhash]-${timestamp}.js`,
+    },
+    module: {
+      rules: [
+        {
+          test: /manifest\.js$/,
+          exclude: /node_modules/,
+          use: {
+            loader: path.resolve(__dirname, 'manifest-loader.js'),
+          },
+        },
+        {
+          test: /\.js$/,
+          exclude: /node_modules/,
+          use: {
+            loader: 'babel-loader',
+            options: {
+              // Speed up compilation.
+              cacheDirectory: '.babelcache',
+              // Also see .babelrc
+            },
+          },
+        },
+        {
+          test: /\.jsx$/,
```

Comment:
We could change this to `/\.jsx?$/` and remove the previous rule, but I decided to leave it for now to show the 1:1 comparison.

Reply:
https://github.com/department-of-veterans-affairs/vets-website/pull/10903 :man_shrugging: